Essentials/src/main/java/com/earth2me/essentials/MobCompat.java

```diff
@@ -34,6 +35,8 @@
     public static final EntityType STRAY = getEntityType("STRAY");
     public static final EntityType FOX = getEntityType("FOX");
     public static final EntityType PHANTOM = getEntityType("PHANTOM");
+    public static final EntityType AXOLOTL = getEntityType("AXOLOTL");
+    public static final EntityType GOAT = getEntityType("GOAT");
```

Comment:
Aren't we missing GLOW_SQUID?

Reply:
`MobCompat` contains constants for use in `MobData`. Since `MobData` has no special entries for glow squids, this is unneeded here.